frontend/src/features/mapSideBar/tabs/propertyDetails/detail/PropertyDetailsTabView.tsx

```diff
@@ -72,13 +72,16 @@ const FormComponent: React.FC<FormikProps<IPropertyDetailsForm>> = ({ values })
     <StyledReadOnlyForm>
       <Section header="Property attributes">
         <SectionField label="MOTI region">
-          <Text field="motiRegion.REGION_NAME" />
+          <Text field="regionType.description" />
         </SectionField>
         <SectionField label="Highways district">
           <InlineContainer>
-            <Text field="highwaysDistrict.DISTRICT_NUMBER" />
-            {'-'}
-            <Text field="highwaysDistrict.DISTRICT_NAME" />
+            {values.districtType?.description !== 'Cannot determine' && (
```

Comment:
nit: would you consider using a constant for 'Cannot determine' ? might help re-use in other places

Reply:
I thought about but given it only is needed on this page decided to leave it as is. We could have a enum for regions in the future